packages/commercetools/api-client/src/api/getOrders/defaultQuery.ts

```diff
@@ -10,6 +10,7 @@ export default gql`
         results {
           ...DefaultOrder
         }
+        total
```

Comment:
What about `offset` and `count` you put in the typescript type below?

Reply:
We are not using it, but I can add it to the query so that it is returned. It is used as a mock value in tests so it would be a good idea to return all fields 👍

Reply:
I will add getters for other properties as well